test/e2e/commit_test.go

```diff
@@ -117,6 +117,31 @@ var _ = Describe("Podman commit", func() {
 		Expect(foundBlue).To(Equal(true))
 	})
 
+	FIt("podman commit container with change CMD flag", func() {
```

Comment:
looks like the CI is complaining this test is taking too long?

Reply:
Extra F on the test here? "It" instead of "FIt"?